### PR TITLE
Fix test discovery in VS for NewtonSoft.Json.Roslyn.sln.

### PR DESCRIPTION
**Pull request by codito**
Add a reference to NUnit3TestAdapter.
Emit full symbols for test project. Workaround for https://github.com/nunit/nunit3-vs-adapter/issues/296.

### CODE REVIEW

File: Src/Newtonsoft.Json.Tests/Newtonsoft.Json.Tests.Roslyn.csproj [hunk 1]
```diff
@@ -93,6 +98,9 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0-beta5-build1225" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0-*" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```
Comment (JamesNK) [inline Src/Newtonsoft.Json.Tests/Newtonsoft.Json.Tests.Roslyn.csproj]: What is the deal with this GUID?

Comment (JonHanna) [inline Src/Newtonsoft.Json.Tests/Newtonsoft.Json.Tests.Roslyn.csproj]: VS likes test projects to have either a particular project type GUID or this service GUID to say "this is an NUnit or XUnit test project, pre-load the test-explorer package".

Comment (JonHanna) [inline Src/Newtonsoft.Json.Tests/Newtonsoft.Json.Tests.Roslyn.csproj]: https://connect.microsoft.com/VisualStudio/feedback/details/800245/vs2013rc-adds-to-vs2012-c-project-section-itemgroup

Comment (codito) [inline Src/Newtonsoft.Json.Tests/Newtonsoft.Json.Tests.Roslyn.csproj]: https://github.com/Microsoft/vstest/issues/472 is tracking removal of this guid.

**Comment by JamesNK** [comment]
Merged. Thanks

**Pull request merged**